From dcd4db9d0f72d98eb2ad1313ba5a8926a6edc66d Mon Sep 17 00:00:00 2001
From: Liu Xiaowen <b37945@freescale.com>
Date: Fri, 9 Aug 2013 15:44:04 +0800
Subject: [PATCH 233/602] ENGR00215174 add ro.bq.gpu_to_cpu_unsupported
 property.

add ro.bq.gpu_to_cpu_unsupported property.
it is a new added property in android 4.3.
and it is used in captureScreen case to allow
gpu and cpu share and access the same memory.
our gpu has the limition to do this.

Signed-off-by: Liu Xiaowen <b37945@freescale.com>
---
 arm2_6dq/init.rc            | 1 +
 arm2_6sl/init.rc            | 1 +
 evk_6sl/init.rc             | 1 +
 hdmidongle_6dq/init.rc      | 1 +
 imx53_smd/init.freescale.rc | 1 +
 sabreauto_6q/init.rc        | 1 +
 sabresd_6dq/init.rc         | 1 +
 7 files changed, 7 insertions(+)

diff --git a/arm2_6dq/init.rc b/arm2_6dq/init.rc
index e267994..dc5a161 100755
--- a/arm2_6dq/init.rc
+++ b/arm2_6dq/init.rc
@@ -17,6 +17,7 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
+    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     # fsl omx graphic manager media framework property
     setprop media.omxgm.enable-player 1
diff --git a/arm2_6sl/init.rc b/arm2_6sl/init.rc
index 646105f..ff5a216 100755
--- a/arm2_6sl/init.rc
+++ b/arm2_6sl/init.rc
@@ -14,6 +14,7 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	0
+    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     # fsl omx graphic manager media framework property
 
diff --git a/evk_6sl/init.rc b/evk_6sl/init.rc
index b786b3a..3980402 100644
--- a/evk_6sl/init.rc
+++ b/evk_6sl/init.rc
@@ -31,6 +31,7 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	0
+    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
diff --git a/hdmidongle_6dq/init.rc b/hdmidongle_6dq/init.rc
index a52334a..a70b3ad 100755
--- a/hdmidongle_6dq/init.rc
+++ b/hdmidongle_6dq/init.rc
@@ -32,6 +32,7 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
+    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
diff --git a/imx53_smd/init.freescale.rc b/imx53_smd/init.freescale.rc
index 0bc9475..f4708e7 100755
--- a/imx53_smd/init.freescale.rc
+++ b/imx53_smd/init.freescale.rc
@@ -40,6 +40,7 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.hw         1
     setprop ro.sf.lcd_density   160
+    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     # fsl omx graphic manager media framework property
     setprop media.omxgm.enable-player 1
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index fdf7065..e264e86 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -32,6 +32,7 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
+    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index de323c6..3bf3d95 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -37,6 +37,7 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
+    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
-- 
1.8.0

